QA/PoroDevTest/pom.xml

```diff
@@ -11,17 +11,36 @@
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
-    </properties>
 
+        <!-- properties needed for Allure -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <aspectj.version>1.9.6</aspectj.version>
+        <selenium-version>4.0.0-beta-3</selenium-version>
+        <testng-version>7.4.0</testng-version>
+        <ashot-version>1.5.4</ashot-version>
```

Comment:
Do we still need ashot dependency related variable in pom file?

Reply:
Moved this line.